recipes-debian/httping/httping_debian.bb

```diff
@@ -0,0 +1,12 @@
+inherit debian-package
+require recipes-debian/sources/httping.inc
+DEBIAN_QUILT_PATCHES = ""
+
+LICENSE = "AGPL-3.0 & OpenSSL"
```

Comment:
I cannot accept httping because license of this program is unclear.
At least, this program doesn't include OpenSSL licensed part but some kind of OpenSSL Exception.
Some files seems GPL-2.0-with-OpenSSL-exception or AGPL-3.0-with-OpenSSL-exception, but other files are unclear.
In addition, original LICENSE file is removed in Debian source package (I don't know why).

Reply:
> but other files are unclear.

license.txt says this software is distributed with AGPL-3.0-with-OpenSSL-exception. So, it is natural that code without license header are also with the license. I do not think it is necessary that all source code file has license header. Many projects has files without license headers.

I fixed the LICENSE= field.